app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update]
+  before_action :set_participating_seller, only: %i[create]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all
+
+    render json: @tickets
+  end
+
+  # GET /tickets/1
+  def show
+    render json: @ticket
+  end
+
+  # POST /tickets
+  def create
+    puts(params[:number_of_tickets])
+    savedCorrectly = true
+    createdTickets = []
+
+    (1..params[:number_of_tickets]).each do |_i|
+      attributes = {}
+      attributes[:ticket_id] = SecureRandom.alphanumeric(5).upcase.insert(4, '-')
+      attributes[:participating_seller] = @participating_seller
+
+      @ticket = Ticket.new(attributes)
+      savedCorrectly &&= @ticket.save
+      createdTickets << @ticket
+    end
+
+    if savedCorrectly
+      render json: createdTickets, status: :created
+    else
+      render 'Error', status: :unprocessable_entity
```

Comment:
Does this really get hit? It should automatically fail and get caught in the exception_handler